public/app/plugins/datasource/prometheus/querybuilder/components/LabelFilterItem.tsx

```diff
@@ -6,9 +6,10 @@ import { selectors } from '@grafana/e2e-selectors';
 import { AccessoryButton, InputGroup } from '@grafana/experimental';
 import { AsyncSelect, Select } from '@grafana/ui';
 
-import { PROMETHEUS_QUERY_BUILDER_MAX_RESULTS } from '../components/MetricSelect';
 import { QueryBuilderLabelFilter } from '../shared/types';
 
+import { PROMETHEUS_QUERY_BUILDER_MAX_RESULTS } from './MetricSelect';
```

Comment:
imported but never used?

Reply:
It is being used. See: https://github.com/grafana/grafana/pull/59749/files#diff-6ec019fb24e14b099249202ed75e8e378f6c68890de1abb139fef56e82ad70f8R124

Reply:
Oh, it was just moved, sorry, I'm blind!